twill-common/src/main/java/org/apache/twill/internal/Constants.java

```diff
@@ -82,6 +82,14 @@ private Files() {
     }
   }
 
+  /**
+   * Constants for logLevel system messages.
+   */
+  public static final class SystemMessages {
```

Comment:
Why need string constants here? All system messages/commands should be defined in `SystemMessages`. You can use object comparison instead of string comparison.